src/HeaderBag.php

```diff
@@ -103,6 +106,6 @@ public function getValue()
 
     public function __toString()
     {
-        return $this->name . ($this->value === '' ? '' : ': ' . $this->value);
+        return $this->name . ':' .(empty($this->value) ? '' : ' ' . $this->value);
```

Comment:
Much like PHPs type hinting – that type assertion function will allow `null` to be passed (so you can see if the user left the default value in, without having to hard code check for the default). SecureHeaders defaults a header value to null if it isn't specified, so it's plausible that the value may be of null type (which should also set a header with no value).